types/react-image-magnify/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // TODO
```

Comment:
do you need this anymore? I didn't see any `| any`, which I think is what this rule forbids.